# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../torcs/depend-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- torcs-1.2.2/Make-default.mk.orig	2005-01-02 23:38:15.724695600 +0100
+++ torcs-1.2.2/Make-default.mk	2005-01-02 23:52:18.544567504 +0100
@@ -134,7 +134,10 @@
 endif
 
 .depend:
-	$(CPP) $(INCFLAGS) $(CFLAGSD) $(COMPILFLAGS)  $(SOURCES) > .depend
+	rm -f .depend; \
+	list="$(SOURCES)" ; for i in $$list; do \
+		$(CPP) -MM $(INCFLAGS) $(CFLAGSD) $(COMPILFLAGS) $$i >> .depend; \
+	done
 
 dep:	.depend
 
